views/components/Graphs/BarGraph.jsx

```diff
@@ -40,6 +41,15 @@ const BarGraph = ({ graph }) => {
                 fill: ({ datum }) => datum.color,
               },
             }}
+            labels={({ datum: { count, studyTarget } }) =>
+              `Current: ${count} \n Target: ${studyTarget}`
```

Comment:
https://github.com/AMP-SCZ/dpdash/pull/117/files#r934763025
It's used here as a template string, I could parse it I don't know if we have to. What do you think?

Reply:
I think it's fine as is. We might want it to be an integer when we need to convert to percentages and factor the target in, but we can cross that bridge when we get to it.